src/test/kotlin/io/github/rybalkinsd/kohttp/performance/KhttpAsyncHttpGetPerformanceTest.kt

```diff
@@ -0,0 +1,99 @@
+package io.github.rybalkinsd.kohttp.performance
+
+import io.github.rybalkinsd.kohttp.dsl.async.httpGetAsync
+import io.github.rybalkinsd.kohttp.ext.url
+import io.github.rybalkinsd.kohttp.getCode
+import io.github.rybalkinsd.kohttp.getSuccessfulResponsesAmount
+import io.github.rybalkinsd.kohttp.pool.CoroutineExecutorService
+import khttp.responses.Response
+import kotlinx.coroutines.*
+import org.junit.Test
+import java.util.concurrent.ForkJoinPool
+import java.util.concurrent.TimeUnit
+import java.util.concurrent.atomic.AtomicInteger
+import kotlin.system.measureTimeMillis
+
+public class KhttpAsyncHttpGetPerformanceTest {
```

Comment:
this test is not related to the release, also better to move to the experimental branch